workspaces/adventure-pack/goodies/kotlin/src/traverse_level_order/Main.kt

```diff
@@ -0,0 +1,26 @@
+package traverse_level_order
+
+import common.TreeNode
+
+public fun TreeNode?.traverseLevelOrder(): Sequence<TreeNode> {
+    if (this == null) return sequenceOf()
```

Comment:
If I recall correctly, I came across a comment in a previous PR suggesting that for this repo we prefer wrapping return statements in braces. Let's check with Miorel if that's the styles we should all adopt.